src/components/MediaPlayerCarouselChapters.tsx

```diff
@@ -64,7 +65,7 @@ export class MediaPlayerCarouselChapters extends React.PureComponent<Props> {
       episode
     }
 
-    return item && item.episode && item.episode.id ? (
+    return item?.episode?.id ? (
```

Comment:
This one comes from the currentChapters. Since it relies on data from our endpoint, I'm not sure we need to guard against this one right now.